rocks/RocksDatabase.java

```diff
@@ -614,15 +614,12 @@ private void countFn() {
                 } catch (TypeDBException e) {
                     if (e.code().isPresent() && e.code().get().equals(DATABASE_CLOSED.code())) {
                         break;
+                    } else if (e.code().isPresent() && (e.code().get().equals(TRANSACTION_CONSISTENCY_MODIFY_DELETE_VIOLATION.code()) ||
```

Comment:
Do we wrap rocksdb exception with our exception now?

Reply:
rocks doesn't throw this exception anymore, we do all the conflict detection ourselves (#6306 )